### DISCUSSION TITLE
Having trouble with historical module throwing HTTPError: Not Found

### DISCUSSION THREAD

**Discussion by ColeTown** [gadicc/yahoo-finance2#405] [General]
I saw in the historical readme that this error generally means that the symbol has been delisted from yahoo, however, I have tried many different symbols, all of which I know are still on yahoo finance. 

The following is my code and the error that I got. The terminal shows the symbols which I was testing. Also, 'loop' is a new Date() variable.

![image](https://user-images.githubusercontent.com/33290898/154885004-c26fed03-5e46-44fb-ba82-35a7498cac8b.png)

I am brand new to using this API, so there is likely something I am missing here, but I can't figure out what it is. 
Any help with this would be greatly appreciated!

**Comment by ColeTown** [comment]
The code in the image shows me using the _chart module instead of the historical module. I have tried both of modules with the same outcome.

**Comment by gadicc** [comment]
Hey @ColeTown, welcome to the lib.

Two issues here:

1) The symbol query parameter takes a single (string) symbol only, not an array of multiple symbols.  I guess we should validate this too before trying the query and I'll add that to our list.  #406

2) You specified the same time for `period1` and `period2`, so you're guaranteeing that you'll never receive any data :)  `period2` is optional and defaults to the current time.  I guess it wouldn't hurt for us to throw an error here too explaining.  #407

So let us know and thanks for your input that helps us make this clearer to others too.
